### PR TITLE
Null pointer exception fix.

### LINKED ISSUE #453: Small bug introduced in 2.2
http://crashes.to/s/084e28f934d

### PR DESCRIPTION
**Pull request by tasomaniac**
Fixes #453 and Crashlytics #138

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java [hunk 1]
```diff
@@ -150,7 +150,10 @@ private void setupDrawerContent(NavigationView navigationView) {
 
         menu.setGroupCheckable(GROUP_ID, true, true);
 
-        menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME)).setChecked(true);
+        MenuItem selectedItem = menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME));
```
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java]: the readability of this is still not really good.

Could you split this up into two lines for getting the string and then finding the item?

Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java]: Can do it but I have to go to bed now. :) I already send a beta update with these merged locally. :) 
Will do the release after all of them merged. We can do it tomorrow.

Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java]: :+1: Thanks!

**Comment by stefanhoth** [comment]
:+1: :shipit:

**Pull request merged**